docs/api/cases/cases-api-create.asciidoc

```diff
@@ -0,0 +1,203 @@
+[[cases-api-create]]
+== Create case API
+++++
+<titleabbrev>Create case</titleabbrev>
+++++
+
+Creates a case.
+
+=== Request
+
+`POST <kibana host>:<port>/api/cases`
+
+=== Prerequisite
+
+You must have `all` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+=== Request body
+
+`connector`::
+(Required, object) An object that contains the connector configuration.
++
+.Properties of `connector`
+[%collapsible%open]
+====
+`fields`::
+(Required, object) An object containing the connector fields.
++
+--
+To create a case without a connector, specify `null`. If you want to omit any
+individual field, specify `null` as its value.
+
+For {sn-itsm} connectors, refer to <<servicenow-action-configuration>>. For
+example:
+
+* `urgency`: The extent to which the incident resolution can delay.
```

Comment:
Would `The extent to which the incident resolution can be delayed.` read better?

Reply:
Thanks!  I've made that change in 17d643b818c8b4d9a020e3bf2b428761cdc30348